dbt_project.yml

```diff
@@ -3,7 +3,7 @@ version: '0.4.0'
 
 config-version: 2
 
-require-dbt-version: ">=1.1.0"
+require-dbt-version: ">=1.3.0"
```

Comment:
Why 1.3.0?

Reply:
ok this means, users need to upgrade to 1.5.0 to use the package. I'll let  @cecile-cecile  weigh on this.

Reply:
@alfredodimassimo On retrospect, let's keep 1.3.0 as the minimum version to use the package. We don't want the users to upgrade just to use this package.